src/PAModel/Utility/Utilities.cs

```diff
@@ -173,8 +173,20 @@ public static string GetRelativePath(string full, string basePath)
         private static string AppendDirectorySeparatorChar(string path)
```

Comment:
Thanks for looking at this. 

Can you add a test case to  UtilityTests.cs!TestRelativePath to cover this case?

Reply:
Added the test case as requested.